stable/shell/templates/shell-configmap.yaml

```
@@ -146,4 +146,12 @@ data:
       echo "# ----- END KUBECTL AUTOCOMPLETION -----" >> ${HOME}/.bashrc
     fi
 
+{{- if .Values.sshServer.enabled }}
+    cp /var/run/iguazio/secrets/ssh/* /igz/.ssh
+    chown iguazio:iguazio /igz/.ssh/*
```

```suggestion
    chown -R iguazio:iguazio ${HOME}/.ssh/
```
How does this work with `/igz/.ssh` :\

---

Why do you chown here? Who is the original owner?
I'm asking as we are about to use other users to execute jupyter (and I guess that shell will follow)

---

And why /igz/.ssh? This is not the users's home dir
The user's home path is ${HOME} as Oded indicated.
If you use it, you have to keep in mind that this is a persistent path on our fuse (I don't know whether you want to override it or not).
If it's there, there's no point in chowning it at all.
Do you want to allow the user to edit those files?
Is the secret here just a way to initialize it?

---

- using $HOME doesn't work - as we don't have users other than _iguazio_.
and the _iguazio_ user's home dir is `/igz` (which is $IGUAZIO_HOME)
- and true - it seems there's no need for chown here
- about editing the public keys, I'm not sure if there's a need/restriction for that - I'll ask product to make sure

---

$HOME should point you to /User which isn't iguazio's home, but the users's fuse home (could be any user).
We manage all the user's stuff there (.bashrc etc.).
Not sure what's the right way to go here, but if you don't use it, there's no persistence anyway (no way to edit the keys outside the secret).